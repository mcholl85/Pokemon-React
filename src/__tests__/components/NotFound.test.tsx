import { render } from '@testing-library/react'
import NotFound from '../../components/NotFound'

describe('NotFound', () => {
  test('Matches DOM snapshot', () => {
    const { asFragment } = render(<NotFound />)

    expect(asFragment()).toMatchSnapshot()
  })
})
