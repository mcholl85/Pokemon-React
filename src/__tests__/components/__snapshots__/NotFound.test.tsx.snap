// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Matches DOM snapshot 1`] = `
<DocumentFragment>
  <div>
    Sorry, no result...
  </div>
</DocumentFragment>
`;
